wgpu-native/src/resource.rs

```diff
@@ -172,6 +173,7 @@ pub struct TextureDescriptor {
 }
 
 pub(crate) enum TexturePlacement<B: hal::Backend> {
+    #[cfg_attr(feature = "remote", allow(unused))]
```

Comment:
Would there be any downside in applying this to the entire wgpu-native crate (for `remote` only)?

Reply:
This is only temporary. Surely we'll want swapchain-backend textures as we progress with the remote implementation. I just wanted to avoid unnecessary warnings now.